src/routes/rootCatalog.ts

```diff
@@ -25,7 +25,7 @@ const selfLinks = (req: Request): Link[] => {
     },
     {
       rel: "service-desc",
-      href: `${stacRoot}/resources/swagger.json`,
+      href: `https://api.stacspec.org/v1.0.0-beta.1/openapi.yaml`,
       title: "OpenAI Documentation",
       type: "application/vnd.oai.openapi+json;version=3.0",
```

Comment:
Should we update the type here since its a yaml file and not json anymore?

Reply:
I'd forgotten about this ticket https://bugs.earthdata.nasa.gov/browse/CMR-10208 I'll try to make sure I'm doing that here

Reply:
The GET request to ` https://api.stacspec.org/v1.0.0-beta.1/openapi.yaml` has a response with `text/yaml` as the `Content-Type`

Reply:
Take a look at this: https://stackoverflow.com/questions/52541842/what-is-the-media-type-of-an-openapi-schema

Still not sure, what it should exactly be though...

Reply:
https://httptoolkit.com/blog/yaml-media-type-rfc/

Reply:
According to IANA the only official media types that I think would be applicable for this yaml file would be:
application/yaml or application/vnd.oai.workflows+yaml
from
https://www.iana.org/assignments/media-types/media-types.xhtml

Reply:
https://datatracker.ietf.org/doc/draft-ietf-httpapi-rest-api-mediatypes/ Looks like this is the latest draft and it is proposing this one:
application/openapi+yaml